Examples/Swift/AppDelegate.swift

```diff
@@ -11,9 +11,15 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
+    #if swift(>=4.0)
+    typealias LaunchOptionsKey = UIApplication.LaunchOptionsKey
+    #else
+    typealias LaunchOptionsKey = UIApplicationLaunchOptionsKey
+    #endif
```

Comment:
I usually only declare the typealias for old versions so that by the time we drop those old version we'll only have to delete the typealias 😉
```
#if !Swift(>=4.0)
extension UIApplication {
  typealias LaunchOptionKey = UIApplicationLaunchOptionKey
}
#endif
```

Reply:
Good point.  Updated.